lib/tasks/sample_data/payment_method_factory.rb

```diff
@@ -33,21 +34,22 @@ def create_cash_method(enterprise)
 
   def create_card_method(enterprise)
     create_payment_method(
+      "Spree::Gateway::Bogus",
       enterprise,
       "Credit card (fake)",
       "We charge 1%, but won't ask for your details. ;-)",
       Spree::Calculator::FlatPercentItemTotal.new(preferred_flat_percent: 1)
     )
   end
 
-  def create_payment_method(enterprise, name, description, calculator)
-    card = enterprise.payment_methods.new(
+  def create_payment_method(provider_class, enterprise, name, description, calculator)
+    payment_method = provider_class.constantize.new(
```

Comment:
What stops us from avoiding the call `constantize` call and just passing constants instead of strings?

Reply:
just my ignorance, I didnt know I could just pass the class directly. done in a new commit.